docs/architecture/architecture_overview.md

```diff
@@ -1,79 +1,133 @@
     end
-    subgraph data_storage[ ]
-        data_storage_label[Data Storage]
-        subgraph bq[<a href='https://console.cloud.google.com/bigquery'>BigQuery</a> - projects]
-            bq_prod[BigQuery prod project]
-            bq_stage[BigQuery staging project]
-        end
-        subgraph gcs[<a href='https://console.cloud.google.com/storage/browser'>GCS</a> - buckets]
-            gcs_other[Metabase, GCP logs, etc.]
-            gcs_payment[Payment buckets]
-            gcs_gtfs[gtfs-data]
-            gcs_gtfs_test[gtfs-data-test]
-        end
-    end
-    subgraph rt_archiver[ ]
-        rt_archiver_label[<a href='https://github.com/cal-itp/data-infra/tree/main/services/gtfs-rt-archive'>RT archiver</a>]
-        prod_rt_archiver[Prod archiver]
-        prepod_rt_archiver[test archiver]
-    end
-    subgraph reports[ ]
-        reports_label[Reports]
-        reports_website[<a href='https://reports.calitp.org'>reports.calitp.org</a>]
-    end
-    subgraph docs[ ]
-        docs_label[Docs]
-        docs_website[<a href='https://docs.calitp.org'>docs.calitp.org</a>]
+    subgraph gcs[<a href='https://console.cloud.google.com/storage/browser'>Google Cloud Storage</a> buckets]
+        gcs_raw[(Raw)]
+        gcs_parsed[(Parsed)]
+        gcs_validation[(Validation)]
+        gcs_analysis[(Analysis artifacts)]
+        gcs_map_tiles[(Map tiles/GeoJSON)]
+        gcs_other[(Backups, Composer code, etc.)]
     end
-    subgraph analysis_tools[ ]
-        analysis_tools_label[Analysis]
+
+    bigquery[(<a href=''>BigQuery</a>)]
+
+    subgraph data_consumers[ ]
+        consumers_label[Data consumers]
         jupyterhub[<a href='https://hubtest.k8s.calitp.jarv.us/hub/'>JupyterHub</a>]
         metabase[<a href='https://dashboards.calitp.org/'>Metabase - dashboards.calitp.org</a>]
+        reports_website[<a href='https://reports.calitp.org'>reports.calitp.org</a>]
     end
-%% links from data sources
-raw_payment --> data_transfer[Data Transfer]
-raw_rt --> rt_archiver
-raw_gtfs --> airflow
+%% data producers
+raw_gtfs -.-> airflow
 airtable --> airflow
-raw_gtfs -.-> schedule_validator
-raw_gtfs -.-> rt_validator
-raw_rt -.-> rt_validator
-%% links from data storage
-bq --> metabase
-bq --> jupyterhub
-bq --> reports
-gcs <--> jupyterhub
-%% links from ungrouped items
-data_transfer --> gcs
-rt_archiver --> gcs
-airflow <--> data_storage
-airflow --> schedule_validator[GTFS Schedule validator  <br> <i>externally maintained</i>]
-airflow --> rt_validator[GTFS RT validator <br> <i>externally maintained</i>]
-schedule_validator --> gcs
-rt_validator --> gcs
+raw_payment -.-> airflow
+airflow ---> gcs_raw
+raw_rt -.-> prod_rt_archiver --> gcs_raw
+raw_rt -.-> test_rt_archiver --> gcs_raw
+%% data transformations
+gcs_raw -->|"GTFS Schedule validator  <br> <i>externally maintained</i>"| gcs_validation
+gcs_raw -->|"GTFS RT validator <br> <i>externally maintained</i>"| gcs_validation
+
```

Comment:
I think there needs to be a link from `gcs_raw` to `gcs_parsed`? Also not sure why some of these connecting lines are orange -- in the current diagram orange means manual (not automated), but I think you may have deleted the key
<img width="897" alt="image" src="https://github.com/cal-itp/data-infra/assets/55149902/4076ff29-b140-43e0-a8ef-1424f32d97c2">